src/modules/10-common/hooks/useGetUsageAndLimit.ts

```diff
@@ -31,6 +31,11 @@ import { useGetCCMLicenseUsage } from 'services/ce'
 import { useLicenseStore } from 'framework/LicenseStore/LicenseStoreContext'
 import { useGetSRMLicenseUsage } from 'services/cv'
 
+export type CHAOSLicenseSummaryDTO = LicensesWithSummaryDTO & {
```

Comment:
this casing can be `ChaosLicenseSummaryDTO` to be consistent with `ChaosModuleLicenseDTO`
```suggestion
export type ChaosLicenseSummaryDTO = LicensesWithSummaryDTO & {
```